python/neuroglancer/lib.py

```diff
@@ -419,19 +452,21 @@ def expand_to_chunk_size(self, chunk_size, offset=Vec(0,0,0, dtype=int)):
     result.maxpt = np.ceil(result.maxpt / chunk_size) * chunk_size 
     return result + offset
 
-  def shrink_to_chunk_size(self, chunk_size):
+  def shrink_to_chunk_size(self, chunk_size, offset=Vec(0,0,0, dtype=int)):
```

Comment:
correct me if i'm wrong here. the offset indicates what coordinate the chunks begin at.  i.e. if min is 2 and offset = 2, then that is the beginning of the first chunk. if this is the case can we change the variable name to `chunk_offset` to be clear it's not some other use of an offset.

@nkemnitz also mentioned that he prefers the word `bias` vs `offset` but either works for me :+1: 

Of course with sufficient documentation this would not be an issue. but adding chunk_offset should make it a bit more self-explanatory